virtcontainers/cgroups_test.go

```diff
@@ -197,3 +197,74 @@ func TestUpdateCgroups(t *testing.T) {
 	err = s.cgroupsDelete()
 	assert.NoError(err)
 }
+
+func TestIsSystemdCgroup(t *testing.T) {
+	assert := assert.New(t)
+
+	assert.False(isSystemdCgroup("/kata/afhts2e5d4g5s"))
+	assert.True(isSystemdCgroup("slice:kata:afhts2e5d4g5s"))
```

Comment:
Would be good to add a few more scenarios like:

- `a:b:c:d` (too many fields)
- `:::` (empty fields)
- `""`
- `:`
- `::`
- `:::`
- `a:b`
- `a:b:`
- `:a:b`
- `@:@:@`
- *et cetera*